### ISSUE TITLE
Fanatec Prosche 911 GT3 RS V2 Wheel

### ISSUE THREAD

**Issue by Wanama78** [gotzl/hid-fanatecff#48]
Hello, I know it's a bit old, I just came across it a few days ago.
Data on the steering wheel has been provided, if anything is missing, I will provide it later.
https://github.com/Wanama78/Prosche-911-GT3-RS-V2-Wheel

I don't know if there's anyone who actually wants to take the time to make a driver, that would be really great.

Thanks in advance for these efforts.

**Comment by gotzl** [comment]
Hi, I had a very brief look into one of the capture files and I do think it looks promissing that it actually also uses the same protocol as all the other wheels. Could you provide the USB PID of the device?

**Comment by Wanama78** [comment]
Hi,

USB PID 0eb7:0197
900° is the impact.

ATS, ETS2, AM2, ACC It works without any problems.

Yes according to the post it seems to be the same thing just with a different name.

**Comment by Wanama78** [comment]
I'm thrilled, it seems to be working well in ETS2 ATS ACC, I've tested it a bit now.
In ETS2, in any case, the FB, sim race works, but you shouldn't expect much (maybe it still will).

Maybe you can adjust a few values ​​in the driver to get more power.
The fan didn't even come on anymore and it didn't get hot either.

It will be used more often now because it is definitely great.

I would like to thank you for your performance here.

**Comment by gotzl** [comment]
> Maybe you can adjust a few values ​​in the driver to get more power.
The fan didn't even come on anymore and it didn't get hot either.

So you're saying, the FFB should be stronger? In game strength is already maxed?

> I would like to thank you for your performance here.

Sure, happy to help :)

**Comment by Wanama78** [comment]
Yes, I'm happy to finally be able to use this steering wheel under Linux, thank you.

**Comment by gotzl** [comment]
So, I had a look at your dumped data. I'm a bit puzzled that it actually works ;) From the dumps I would've said that the protocol should be a bit different.. But maybe it understands both? Anyways, maybe you can provide some more data of other effects, as I've suggested in the ticket in your data-dump repo...

**Comment by Wanama78** [comment]
Yes, I will make it.

Well, maybe that's why it's a bit flat that FFB that remains to be seen.

I'm only using one VM here for Win, I hope that I can continue to read everything well with it.
If you now think a physical Windows would be better then I would have to set up a complete system, only when really needed.

**Comment by gotzl** [comment]
Closing this as of #163
